libs/core/src/schemas/entities.schemas.ts

```diff
@@ -491,3 +495,40 @@ export const Outbox = z.object({
   created_at: z.date(),
   updated_at: z.date(),
 });
+
+export const SubscriptionPreference = z.object({
+  id: z.number(),
+  user_id: z.number(),
+  email_notifications_enabled: z.boolean().default(false),
+  digest_email_enabled: z.boolean().default(false),
+  recap_email_enabled: z.boolean().default(false),
+  mobile_push_notifications_enabled: z.boolean().default(false),
+  mobile_push_discussion_activity_enabled: z.boolean().default(false),
+  mobile_push_admin_alerts_enabled: z.boolean().default(false),
+  created_at: z.date().default(new Date()),
+  updated_at: z.date().default(new Date()),
+});
+
+export const ThreadSubscription = z.object({
+  id: z.number(),
+  user_id: z.number(),
+  thread_id: z.number(),
```

Comment:
PG_INT?

Reply:
- `id` is generated by the DB
- `user_id` is a foreign key so it always has to be defined
- `thread_id` is also a foreign key

None of these values will or should be mocked in the DB so min/max is unnecessary. That said I will add `int()`

Reply:
@rbennettcw can give you some extra context, but he started adding min/max range to int cols to protect the db from out of range fields in payloads ... encapsulating this rule in something like PG_INT shouldn't hurt

Reply:
Makes sense.